tensorrt_llm/_torch/attention_backend/interface.py

```
@@ -140,6 +140,7 @@ class AttentionMetadata:
 
     _saved_tensors: Dict[str, torch.Tensor] = field(init=False,
                                                     default_factory=dict)
+    sparse_attention_config: Optional["SparseAttentionConfig"] = None
```

Ruff (0.13.1)

Undefined name `SparseAttentionConfig`

(F821)